config/server/server.js

```diff
@@ -19,6 +19,10 @@ if (middleware) {
 }
 app.get('*', renderCallback);
 
-app.listen(port, () => {
-  console.log(`App started on port ${port}`);
-});
+if (env === 'production') {
+  app.listen(port, () => {
```

Comment:
This port also gets listened to in server/index right?  Why is this necessary?

Reply:
server/index is only for dev. This is for production

Reply:
Which implied you found a bug :|